### PR TITLE
Chat 2: Part 7 - conversation messages

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: shared/chat/conversation/messages/error/index.js [hunk 1]
```diff
@@ -1,28 +1,34 @@
 // @flow
-import * as React from 'react'
-import {Text, Box} from '../../../../common-adapters'
-import {globalStyles, globalColors} from '../../../../styles'
+// import * as React from 'react'
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/error/index.js]: i'll fix this in branch 10

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: shared/chat/conversation/messages/error/index.js (hunk 1 above)
Comment (chrisnojima) [reply shared/chat/conversation/messages/error/index.js]: fixed

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/messages/wrapper/container.js [hunk 2]
```diff
@@ -1,140 +1,117 @@
...
+    hasOlderResetConversation: stateProps.hasOlderResetConversation,
     includeHeader,
-    innerClass: ownProps.innerClass,
+    innerClass: stateProps.innerClass,
     isBroken: stateProps.isBroken,
-    isEdited,
+    isEdited: message.hasBeenEdited,
     isEditing: stateProps.isEditing,
-    isFirstNewMessage,
+    isFirstNewMessage: false, //  TODO
```
Comment (akalin-keybase) [inline shared/chat/conversation/messages/wrapper/container.js]: fixed in a future PR?

Comment (chrisnojima) [reply shared/chat/conversation/messages/wrapper/container.js]: yeah its actually removed. the orange line handling is done another way and this was just a leftover

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/messages/index.js [hunk 3]
```diff
@@ -1,80 +1,151 @@
...
+        return <SystemGitPush message={this.props.message} />
+      case 'systemAddedToTeam':
+        return <SystemAddedToTeam message={this.props.message} />
+      case 'systemJoined':
+        return <SystemJoined message={this.props.message} />
+      case 'systemText':
+        return <SystemText message={this.props.message} />
+      case 'systemLeft':
+        return <SystemLeft message={this.props.message} />
+      // case 'error':
```
Comment (mmaxim) [inline shared/chat/conversation/messages/index.js]: What's up with this?

Comment (chrisnojima) [reply shared/chat/conversation/messages/index.js]: this is fixed in part 10. errors are just treated as type Text w/ an error Reason now. there's no specific error type anymore

**Pull request merged**